package.json

```diff
@@ -181,7 +181,8 @@
       "@typescript-eslint/parser": "^5.52.0"
     },
     "patchedDependencies": {
-      "babel-plugin-typescript-to-proptypes@1.4.2": "patches/babel-plugin-typescript-to-proptypes@1.4.2.patch"
+      "babel-plugin-typescript-to-proptypes@1.4.2": "patches/babel-plugin-typescript-to-proptypes@1.4.2.patch",
+      "vite-bundle-analyzer@0.12.1": "patches/vite-bundle-analyzer@0.12.1.patch"
```

Comment:
`vite-bundle-analyzer` has a `rollup` very flexible peer dependency (2.X -> 4.X) but it actually requires 3.X at least because of a typescript type usage.
Since we're using rollup 2.X in the repo (force by `rollup`), the alternative I took is to change the type usage (very simple change)